ui/app/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -384,6 +386,33 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
+  renderLedgerLiveControl() {
+    const { t } = this.context;
+    const { useLedgerLive, setLedgerLivePreference } = this.props;
+
+    return (
+      <div className="settings-page__content-row">
+        <div className="settings-page__content-item">
+          <span>{t('ledgerLiveAdvancedSetting')}</span>
+          <div className="settings-page__content-description">
+            {t('ledgerLiveAdvancedSettingDescription')}
+          </div>
+        </div>
+        <div className="settings-page__content-item">
+          <div className="settings-page__content-item-col">
+            <ToggleButton
+              value={useLedgerLive}
+              onToggle={(value) => setLedgerLivePreference(!value)}
+              offLabel={t('off')}
+              onLabel={t('on')}
+              disabled={!('usb' in window.navigator)}
```

Comment:
What is the connection between the Ledger Live bridge and WebUSB? I thought one of the main advantages of this was that it didn't depend on WebUSB, so it would work on Firefox.

Reply:
Good shout -- removed!